MSVC build fails due to cub symlink: fatal error C1083: Cannot open include file: '../config.cuh': No such file or directory

With a freshly cloned thrust (and recursively cloned CUB) i'm getting 'file not found' errors in CUB:

```
  Compiling CUDA source file headers\thrust\adjacent_difference.h.cu...

  F:\code\thrust\build>"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\nvcc.exe" -gencode=arch=compute_80
  ,code=\"compute_80,compute_80\" -gencode=arch=compute_35,code=\"sm_35,compute_35\" -gencode=arch=compute_37,code=\"sm
  _37,compute_37\" -gencode=arch=compute_50,code=\"sm_50,compute_50\" -gencode=arch=compute_52,code=\"sm_52,compute_52\
  " -gencode=arch=compute_53,code=\"sm_53,compute_53\" -gencode=arch=compute_60,code=\"sm_60,compute_60\" -gencode=arch
  =compute_61,code=\"sm_61,compute_61\" -gencode=arch=compute_62,code=\"sm_62,compute_62\" -gencode=arch=compute_70,cod
  e=\"sm_70,compute_70\" -gencode=arch=compute_72,code=\"sm_72,compute_72\" -gencode=arch=compute_75,code=\"sm_75,compu
  te_75\" -gencode=arch=compute_80,code=\"sm_80,compute_80\" --use-local-env -ccbin "F:\Program Files (x86)\Microsoft V
  isual Studio\2019\Preview\VC\Tools\MSVC\14.28.29331\bin\HostX64\x64" -x cu   -IF:\code\thrust -IF:\code\thrust\depend
  encies\cub -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\include"     --keep-dir x64\Debug -maxrregcoun
  t=0  --machine 64 --compile -cudart static -Xcudafe=--display_error_number -Wno-deprecated-gpu-targets -Xcudafe=--pro
  mote_warnings -Xcompiler="/EHsc -Zi -Ob0 /WX /wd4244 /wd4267 /wd4800 /wd4146 /wd4494 /bigobj" -g   -D_WINDOWS -DTHRUS
  T_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -DNOMINMAX -D"CMAKE_INTDIR=\"De
  bug\"" -D"CMAKE_INTDIR=\"Debug\"" -D_MBCS -Xcompiler "/EHsc /W3 /nologo /Od /Fdthrust.headers.dir\Debug\thrust.header
  s.pdb /FS /Zi /RTC1 /MDd /GR" -o thrust.headers.dir\Debug\adjacent_difference.h.obj "F:\code\thrust\build\headers\thr
  ust\adjacent_difference.h.cu"
F:\code\thrust\cub\block\block_exchange.cuh(36): fatal error C1083: Cannot open include file: '../config.cuh': No such
file or directory [F:\code\thrust\build\thrust.headers.vcxproj]
  adjacent_difference.h.cu
F:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Microsoft\VC\v160\BuildCustomizations\CUDA 11.0.tar
gets(772,9): error MSB3721: The command ""C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin\nvcc.exe" -genco
de=arch=compute_80,code=\"compute_80,compute_80\" -gencode=arch=compute_35,code=\"sm_35,compute_35\" -gencode=arch=comp
ute_37,code=\"sm_37,compute_37\" -gencode=arch=compute_50,code=\"sm_50,compute_50\" -gencode=arch=compute_52,code=\"sm_
52,compute_52\" -gencode=arch=compute_53,code=\"sm_53,compute_53\" -gencode=arch=compute_60,code=\"sm_60,compute_60\" -
gencode=arch=compute_61,code=\"sm_61,compute_61\" -gencode=arch=compute_62,code=\"sm_62,compute_62\" -gencode=arch=comp
ute_70,code=\"sm_70,compute_70\" -gencode=arch=compute_72,code=\"sm_72,compute_72\" -gencode=arch=compute_75,code=\"sm_
75,compute_75\" -gencode=arch=compute_80,code=\"sm_80,compute_80\" --use-local-env -ccbin "F:\Program Files (x86)\Micro
soft Visual Studio\2019\Preview\VC\Tools\MSVC\14.28.29331\bin\HostX64\x64" -x cu   -IF:\code\thrust -IF:\code\thrust\de
pendencies\cub -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\include"     --keep-dir x64\Debug -maxrregco
unt=0  --machine 64 --compile -cudart static -Xcudafe=--display_error_number -Wno-deprecated-gpu-targets -Xcudafe=--pro
mote_warnings -Xcompiler="/EHsc -Zi -Ob0 /WX /wd4244 /wd4267 /wd4800 /wd4146 /wd4494 /bigobj" -g   -D_WINDOWS -DTHRUST_
HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -DNOMINMAX -D"CMAKE_INTDIR=\"Debug\
"" -D"CMAKE_INTDIR=\"Debug\"" -D_MBCS -Xcompiler "/EHsc /W3 /nologo /Od /Fdthrust.headers.dir\Debug\thrust.headers.pdb
/FS /Zi /RTC1 /MDd /GR" -o thrust.headers.dir\Debug\adjacent_difference.h.obj "F:\code\thrust\build\headers\thrust\adja
cent_difference.h.cu"" exited with code 2. [F:\code\thrust\build\thrust.headers.vcxproj]
```

Here is the configure output:
```
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19041.
-- The CXX compiler identification is MSVC 19.28.29331.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: F:/Program Files (x86)/Microsoft Visual Studio/2019/Preview/VC/Tools/MSVC/14.28.29331/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Thrust: F:/code/thrust/thrust/cmake/thrust-config.cmake (found version "1.11.0.0")
-- Found CUB: F:/code/thrust/dependencies/cub/cub/cmake/cub-config.cmake (found version "1.11.0.0")
-- Thrust: TargetInfo: thrust: (1.11.0.0)
-- Thrust: TargetInfo: thrust > IMPORTED: TRUE
-- Thrust: TargetInfo: thrust > INTERFACE_LINK_LIBRARIES: Thrust::CPP::Host;Thrust::CUDA::Device
-- Performing Test CXX_FLAG__W3
-- Performing Test CXX_FLAG__W3 - Success
-- Performing Test CXX_FLAG__WX
-- Performing Test CXX_FLAG__WX - Success
-- Performing Test CXX_FLAG__wd4244
-- Performing Test CXX_FLAG__wd4244 - Success
-- Performing Test CXX_FLAG__wd4267
-- Performing Test CXX_FLAG__wd4267 - Success
-- Performing Test CXX_FLAG__wd4800
-- Performing Test CXX_FLAG__wd4800 - Success
-- Performing Test CXX_FLAG__wd4146
-- Performing Test CXX_FLAG__wd4146 - Success
-- Performing Test CXX_FLAG__wd4494
-- Performing Test CXX_FLAG__wd4494 - Success
-- Performing Test CXX_FLAG__bigobj
-- Performing Test CXX_FLAG__bigobj - Success
-- Enabling Thrust configuration: cpp.cuda.cpp14
-- CPP system found?  TRUE
-- CUDA system found? TRUE
-- TBB system found?  FALSE
-- OMP system found?  FALSE
-- The CUDA compiler identification is NVIDIA 11.0.221
-- Detecting CUDA compiler ABI info
-- Detecting CUDA compiler ABI info - done
-- Check for working CUDA compiler: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.0/bin/nvcc.exe - skipped
-- Detecting CUDA compile features
-- Detecting CUDA compile features - done
-- Thrust: Enabled CUDA architectures: sm_35 sm_37 sm_50 sm_52 sm_53 sm_60 sm_61 sm_62 sm_70 sm_72 sm_75 sm_80 compute_80
sm_53 does not support RDC. Targets that require RDC will be built without support for this architecture.
sm_62 does not support RDC. Targets that require RDC will be built without support for this architecture.
sm_72 does not support RDC. Targets that require RDC will be built without support for this architecture.
-- Configuring done
-- Generating done
-- Build files have been written to: F:/code/thrust/build
```

I have a same issue when using thrust with CMake on GitHub Action windows-2019 virtual machine.

Logs:
````
2021-07-23T12:20:44.0344123Z ##[group]Run $trimeshDir = "..\trimesh2-build"
2021-07-23T12:20:44.0344888Z [36;1m$trimeshDir = "..\trimesh2-build"[0m
2021-07-23T12:20:44.0345317Z [36;1mcmake -A x64 `[0m
2021-07-23T12:20:44.0345924Z [36;1m-DCMAKE_TOOLCHAIN_FILE:FILEPATH="C:\vcpkg\scripts\buildsystems\vcpkg.cmake" `[0m
2021-07-23T12:20:44.0346693Z [36;1m-DTrimesh2_INCLUDE_DIR:PATH="$trimeshDir\include" `[0m
2021-07-23T12:20:44.0347349Z [36;1m-DTrimesh2_LINK_DIR:PATH="$trimeshDir\lib.Win64.vs142" `[0m
2021-07-23T12:20:44.0347944Z [36;1m-DCUDA_ARCH:STRING=$env:CUDA_ARCH `[0m
2021-07-23T12:20:44.0348498Z [36;1m-DCMAKE_BUILD_TYPE=Release `[0m
2021-07-23T12:20:44.0349098Z [36;1m-DThrust_DIR=D:\a\cuda_voxelizer\cuda_voxelizer\..\thrust-repo\thrust\cmake `[0m
2021-07-23T12:20:44.0349646Z [36;1m-S . -B .\build[0m
2021-07-23T12:20:44.0392387Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2021-07-23T12:20:44.0392888Z env:
2021-07-23T12:20:44.0393266Z   CUDA_MAJOR_VERSION: 11.3
2021-07-23T12:20:44.0393740Z   CUDA_PATCH_VERSION: 1
2021-07-23T12:20:44.0394157Z   TRIMESH_VERSION: 2020.03.04
2021-07-23T12:20:44.0394513Z   CUDAARCHS: 60
2021-07-23T12:20:44.0394922Z   NVIDIA_TRUST_VERSION: cuda-11.3
2021-07-23T12:20:44.0395478Z   CUDA_PATH: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3
2021-07-23T12:20:44.0396184Z   CUDA_PATH_V11_3: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3
2021-07-23T12:20:44.0396738Z   CUDA_PATH_VX_Y: CUDA_PATH_V11_3
2021-07-23T12:20:44.0397107Z ##[endgroup]
2021-07-23T12:20:44.4055037Z -- Building for: Visual Studio 16 2019
2021-07-23T12:20:56.5812233Z -- The CXX compiler identification is MSVC 19.29.30038.1
2021-07-23T12:21:02.5776535Z -- The CUDA compiler identification is NVIDIA 11.3.109
2021-07-23T12:21:02.6732984Z -- Detecting CXX compiler ABI info
2021-07-23T12:21:05.0103549Z -- Detecting CXX compiler ABI info - done
2021-07-23T12:21:05.0127822Z -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30037/bin/Hostx64/x64/cl.exe - skipped
2021-07-23T12:21:05.0133657Z -- Detecting CXX compile features
2021-07-23T12:21:05.0160687Z -- Detecting CXX compile features - done
2021-07-23T12:21:05.0333761Z -- Detecting CUDA compiler ABI info
2021-07-23T12:21:08.0203734Z -- Detecting CUDA compiler ABI info - done
2021-07-23T12:21:08.0539020Z -- Check for working CUDA compiler: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.3/bin/nvcc.exe - skipped
2021-07-23T12:21:08.0543309Z -- Detecting CUDA compile features
2021-07-23T12:21:08.0549805Z -- Detecting CUDA compile features - done
2021-07-23T12:21:11.0095399Z -- Found OpenMP_CXX: -openmp (found version "2.0") 
2021-07-23T12:21:11.0100286Z -- Found OpenMP: TRUE (found version "2.0")  
2021-07-23T12:21:11.0831989Z -- Found CUDAToolkit: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.3/include (found version "11.3.109") 
2021-07-23T12:21:15.4678404Z -- Found Trimesh2 include: D:/a/cuda_voxelizer/trimesh2-build/include/TriMesh.h
2021-07-23T12:21:15.4679974Z -- Found Trimesh2 lib: D:/a/cuda_voxelizer/trimesh2-build/lib.Win64.vs142/trimesh.lib
2021-07-23T12:21:15.4829470Z -- Found Thrust: D:/a/cuda_voxelizer/thrust-repo/thrust/cmake/thrust-config.cmake (found version "1.11.0.0") 
2021-07-23T12:21:15.4888228Z -- Found CUB: D:/a/cuda_voxelizer/thrust-repo/dependencies/cub/cub/cmake/cub-config.cmake (found version "1.11.0.0") 
2021-07-23T12:21:15.4911045Z -- Configuring done
2021-07-23T12:21:15.5498164Z -- Generating done
2021-07-23T12:21:15.5505602Z CMake Warning:
2021-07-23T12:21:15.5506426Z   Manually-specified variables were not used by the project:
2021-07-23T12:21:15.5507059Z 
2021-07-23T12:21:15.5507554Z     CUDA_ARCH
2021-07-23T12:21:15.5507929Z 
2021-07-23T12:21:15.5508271Z 
2021-07-23T12:21:15.5516155Z -- Build files have been written to: D:/a/cuda_voxelizer/cuda_voxelizer/build



2021-07-23T12:21:15.7235016Z ##[group]Run cmake --build .\build --parallel 2 --target ALL_BUILD --config Release
2021-07-23T12:21:15.7235916Z [36;1mcmake --build .\build --parallel 2 --target ALL_BUILD --config Release[0m
2021-07-23T12:21:15.7276617Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2021-07-23T12:21:15.7277144Z env:
2021-07-23T12:21:15.7277514Z   CUDA_MAJOR_VERSION: 11.3
2021-07-23T12:21:15.7277954Z   CUDA_PATCH_VERSION: 1
2021-07-23T12:21:15.7278332Z   TRIMESH_VERSION: 2020.03.04
2021-07-23T12:21:15.7278728Z   CUDAARCHS: 60
2021-07-23T12:21:15.7279157Z   NVIDIA_TRUST_VERSION: cuda-11.3
2021-07-23T12:21:15.7279712Z   CUDA_PATH: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3
2021-07-23T12:21:15.7280408Z   CUDA_PATH_V11_3: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3
2021-07-23T12:21:15.7280974Z   CUDA_PATH_VX_Y: CUDA_PATH_V11_3
2021-07-23T12:21:15.7281362Z ##[endgroup]
2021-07-23T12:21:16.1803944Z Microsoft (R) Build Engine version 16.10.2+857e5a733 for .NET Framework
2021-07-23T12:21:16.1805376Z Copyright (C) Microsoft Corporation. All rights reserved.
2021-07-23T12:21:16.1805954Z 
2021-07-23T12:21:16.6176867Z   Checking Build System
2021-07-23T12:21:16.7999109Z   Building Custom Rule D:/a/cuda_voxelizer/cuda_voxelizer/CMakeLists.txt
2021-07-23T12:21:17.6273841Z   Compiling CUDA source file ..\src\voxelize.cu...
2021-07-23T12:21:18.7586958Z   
2021-07-23T12:21:18.7608519Z   D:\a\cuda_voxelizer\cuda_voxelizer\build>"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\bin\nvcc.exe" -gencode=arch=compute_60,code=\"compute_60,compute_60\" -gencode=arch=compute_60,code=\"sm_60,compute_60\" --use-local-env -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64" -x cu   -I"D:\a\cuda_voxelizer\cuda_voxelizer\..\trimesh2-build\include" -I"D:\a\cuda_voxelizer\thrust-repo" -I"D:\a\cuda_voxelizer\thrust-repo\dependencies\cub" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\include" -I"C:\vcpkg\installed\x64-windows\include" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\include"     --keep-dir x64\Release  -maxrregcount=0  --machine 64 --compile -cudart static -std=c++17 -Xcompiler="/EHsc -Ob2"   -D_WINDOWS -DNDEBUG -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -D"CMAKE_INTDIR=\"Release\"" -D_MBCS -DWIN32 -D_WINDOWS -DNDEBUG -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -D"CMAKE_INTDIR=\"Release\"" -Xcompiler "/EHsc /W1 /nologo /O2 /Fdcuda_voxelizer.dir\Release\vc142.pdb /FS   /MD " -o cuda_voxelizer.dir\Release\voxelize.obj "D:\a\cuda_voxelizer\cuda_voxelizer\src\voxelize.cu" 
2021-07-23T12:21:18.7613542Z   Compiling CUDA source file ..\src\thrust_operations.cu...
2021-07-23T12:21:18.8483861Z   
2021-07-23T12:21:18.8501921Z   D:\a\cuda_voxelizer\cuda_voxelizer\build>"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\bin\nvcc.exe" -gencode=arch=compute_60,code=\"compute_60,compute_60\" -gencode=arch=compute_60,code=\"sm_60,compute_60\" --use-local-env -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64" -x cu   -I"D:\a\cuda_voxelizer\cuda_voxelizer\..\trimesh2-build\include" -I"D:\a\cuda_voxelizer\thrust-repo" -I"D:\a\cuda_voxelizer\thrust-repo\dependencies\cub" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\include" -I"C:\vcpkg\installed\x64-windows\include" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\include"     --keep-dir x64\Release  -maxrregcount=0  --machine 64 --compile -cudart static -std=c++17 -Xcompiler="/EHsc -Ob2"   -D_WINDOWS -DNDEBUG -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -D"CMAKE_INTDIR=\"Release\"" -D_MBCS -DWIN32 -D_WINDOWS -DNDEBUG -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -D"CMAKE_INTDIR=\"Release\"" -Xcompiler "/EHsc /W1 /nologo /O2 /Fdcuda_voxelizer.dir\Release\vc142.pdb /FS   /MD " -o cuda_voxelizer.dir\Release\thrust_operations.obj "D:\a\cuda_voxelizer\cuda_voxelizer\src\thrust_operations.cu" 
2021-07-23T12:21:18.8531531Z D:\a\cuda_voxelizer\thrust-repo\cub\block\block_exchange.cuh(36): fatal error C1083: Cannot open include file: '../config.cuh': No such file or directory [D:\a\cuda_voxelizer\cuda_voxelizer\build\cuda_voxelizer.vcxproj]
2021-07-23T12:21:18.8564925Z   thrust_operations.cu
````

--- a/cub
+++ b/cub
@@ -1,1 +1,0 @@
-dependencies/cub/cub

